pkg/services/rpcsrv/server.go

```diff
@@ -2185,7 +2184,16 @@ func (s *Server) prepareInvocationContext(t trigger.Type, script []byte, contrac
 
 		err = s.chain.InitVerificationContext(ic, contractScriptHash, &transaction.Witness{InvocationScript: script, VerificationScript: []byte{}})
 		if err != nil {
-			return nil, neorpc.NewInternalServerError(fmt.Sprintf("can't prepare verification VM: %s", err))
+			switch {
+			case errors.Is(err, core.ErrUnknownVerificationContract):
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrUnknownContract, err.Error())
+			case errors.Is(err, core.ErrInvalidVerificationContract):
+				return nil, neorpc.WrapErrorWithData(neorpc.ErrInvalidVerificationFunction, fmt.Sprintf("Missing verify method: %s", err.Error()))
```

Comment:
1. Don't specify `Missing verify method:`, just use the `err` as an argument, because `err` contains everything that needed for the response: `ErrInvalidVerificationContract = errors.New("verification contract is missing verify method")`.

2. The text of `core.ErrInvalidVerificationContract` can be adjusted: `verification contract is missing `verify` method or `verify` method has unexpected return value`. You can add a separate commit for this adjustment to the end of PR.